TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/util/NettyUtils.java

```diff
@@ -0,0 +1,22 @@
+package org.opensearch.migrations.replay.util;
+
+import com.google.errorprone.annotations.MustBeClosed;
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+import java.util.Collection;
+
+import java.util.stream.Stream;
+
+public final class NettyUtils {
+    @MustBeClosed
+    public static Stream<ByteBuf> createCloseableByteBufStream(Stream<byte[]> byteArrStream) {
```

Comment:
Having been separated from this code for a few days, I couldn't remember exactly what needed to be done by this function.  Does it make sense to add something to the name about 'refCnt neutral' or anything else that would indicate, I'll make it & own its destruction, provided that you do indeed close it.

Reply:
Updated to `createRefCntNeutralCloseableByteBufStream`